components/socialNetworks/socialNetworks.module.scss

```diff
@@ -24,6 +23,7 @@
       height: 100%;
     }
     &__icon {
+      padding: 4px;
```

Comment:
Adding to this change, please change the max-width and max-height of the "SocialButton" class to 48px in mobile.

Reply:
Done